etna/reconciliation/base.py

```diff
@@ -83,18 +85,29 @@ def reconcile(self, ts: TSDataset) -> TSDataset:
         current_level_segments = ts.hierarchical_structure.get_level_segments(level_name=self.source_level)
         target_level_segments = ts.hierarchical_structure.get_level_segments(level_name=self.target_level)
 
+        target_names = tuple(get_target_with_quantiles(columns=ts.columns))
```

Comment:
We have already seen it in `_get_dataframe_level`.

Reply:
Fixed